.travis.yml

```diff
@@ -12,7 +12,7 @@ jobs:
   include:
     - stage: test
 
-    - elixir: '1.5.0'
+    - elixir: '1.7.4'
```

Comment:
note: I picked the [same 1.7 version as plug](https://github.com/elixir-plug/plug/blob/master/.github/workflows/ci.yml#L18)